extensions/wikia/RWEPageHeader/scripts/rwe-page-header.js

```diff
@@ -9,27 +9,59 @@ require(['wikia.window', 'jquery', 'wikia.tracker'], function (window, $, tracke
 		}, data));
 	};
 
-	$(function () {
-		var $globalNav = $('.wds-global-navigation-wrapper'),
-			$notificationsWrapper = $('.banner-notifications-wrapper');
-
-		$('.rwe-page-header-nav__link, .rwe-page-header-wordmark_wrapper').on('click', function (e) {
+	function addWordmarkTracking() {
+		$('.rwe-page-header-nav__link, .rwe-page-header-wordmark_wrapper').on('click', function () {
 			track({
 				label: $(this).data().tracking
 			});
 		});
+	}
+
+	function openChatWindowOnClick() {
+		$('.rwe-chat').on('click', function (e) {
+			e.preventDefault();
+			window.ChatWidget.onClickChatButton($(this).attr('href'));
+		});
+	}
 
+	function initReadDropdown() {
+		var firstLevelItems = $('.rwe-page-header-nav__dropdown-first-level-item'),
+			secondLevelItems = $('.rwe-page-header-nav__dropdown-second-level-item');
 
 		$('.rwe-page-header-nav__element-dropdown > .rwe-page-header-nav__link').on('click', function (e) {
 			e.preventDefault();
 		});
 
+		$('.rwe-page-header-nav__dropdown-first-level-item:first-child').addClass('item-selected');
+		$('.rwe-page-header-nav__dropdown-second-level')
+			.find('.rwe-page-header-nav__dropdown-second-level-item:has(ul):first').addClass('item-selected');
+
+		firstLevelItems.hover(function () {
+			var self = $(this),
```

Comment:
using self to store a jQuery object is confusing. I'd either prefix it to convey that in the var name, or assign purely `this` here and wrap in jQuery in L45